From: Leonard Crestez <leonard.crestez@nxp.com>
Date: Thu, 19 Dec 2019 17:42:47 +0200
Subject: PM / devfreq: imx8m-ddrc: Remove unused defines
Git-commit: 15e4e74ea899515bf4eaf6d400d0c04f5433f15a
Patch-mainline: v5.6-rc1
References: jsc#SLE-16106

The IMX_SIP_DDR_DVFS_WAIT_CHANGE and IMX_SIP_DDR_FREQ_SET_HIGH defines
are not used so they can be removed.

Fixes: 518e99e2a22e ("PM / devfreq: Add dynamic scaling for imx8m ddr controller")
Reported-by: Shawn Guo <shawn.guo@kernel.org>
Signed-off-by: Leonard Crestez <leonard.crestez@nxp.com>
Signed-off-by: Chanwoo Choi <cw00.choi@samsung.com>
Acked-by: Nicolas Saenz Julienne <nsaenzjulienne@suse.de>
---
 drivers/devfreq/imx8m-ddrc.c | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/drivers/devfreq/imx8m-ddrc.c b/drivers/devfreq/imx8m-ddrc.c
index 01ff9afdc8bf..53df7923d893 100644
--- a/drivers/devfreq/imx8m-ddrc.c
+++ b/drivers/devfreq/imx8m-ddrc.c
@@ -15,12 +15,6 @@
 
 #define IMX_SIP_DDR_DVFS			0xc2000004
 
-/* Values starting from 0 switch to specific frequency */
-#define IMX_SIP_DDR_FREQ_SET_HIGH		0x00
-
-/* Deprecated after moving IRQ handling to ATF */
-#define IMX_SIP_DDR_DVFS_WAIT_CHANGE		0x0F
-
 /* Query available frequencies. */
 #define IMX_SIP_DDR_DVFS_GET_FREQ_COUNT		0x10
 #define IMX_SIP_DDR_DVFS_GET_FREQ_INFO		0x11

